.github/workflows/test.yml

```diff
@@ -295,6 +295,67 @@ jobs:
           name: logs-test-protectedmode-${{ matrix.test }}
           path: /tmp/logs
 
+  test_backend_azure_blob:
+    name: Test backend with Azure Blob Storage
+    runs-on: ubuntu-latest
+    needs: [build]
+    strategy:
+      matrix:
+        test:
+          - upload1 upload2 upload3 upload4 download
+          # TODO(Ashwin): enable more tests below when we add support for downloading bundle dependencies from Blob Storage.
+          # - unittest gen-rest-docs gen-cli-docs gen-readthedocs basic auth status batch anonymous competition unicode rest1 upload1 upload2 upload3 upload4 download refs binary rm make worksheet_search worksheet_tags freeze detach perm search_time groups
+          # - run
+          # - run2
+          # - search link read kill write mimic workers edit_user sharing_workers
+          # - resources
+          # - memoize
+          # - copy netcat netcurl
+          # - edit
+          # - open wopen
+    steps:
+      - name: Clear free space
+        run: |
+            sudo rm -rf /opt/ghc
+            df -h
```

Comment:
why do we run `df -h` here? do we need the output?

Reply:
ah well it exists in the rest of the test setup as well

Reply:
Yeah, don't think it's needed, but maybe we can remove in a later refactor